### PR TITLE
Add some helpers to Guild

### PR DESCRIPTION
**Pull request by jonas747**
I find myself typing these a lot

I didn't include any for members or channels since thats less rarely used (state has it's own member and channel map)

### CODE REVIEW

**Comment by jonas747** [comment]
Added helpers for members and channels anyways

**Comment by iopred** [comment]
Just nitting on the function names.

**Comment by jonas747** [comment]
How do you mean?

**Comment by NamedKitten** [comment]
I would love to see this be merged.

**Comment by jonas747** [comment]
Done

File: structs.go [hunk 1]
```diff
@@ -241,6 +241,72 @@ type Guild struct {
...
+	for _, m := range g.Members {
+		if m.User.ID == userID {
+			return m
+		}
+	}
+
+	return nil
+}
+
+// Channel returns a channel from the Guild.Channels slice with the given channlID
```
Comment (iopred) [inline structs.go]: nit: you spelled channelID as channlID.

Super nit: All these comments should end in a period.

**Comment by jonas747** [comment]
Also done

**Pull request closed without merging**